06_distributed_advanced.ipynb

```diff
@@ -542,11 +542,11 @@
     "\n",
     "@delayed\n",
     "def summation(*a):\n",
-    "    return sum(*a)\n",
+    "    return sum(a)\n",
     "\n",
     "ina = [5, 25, 30]\n",
     "inb = [5, 5, 6]\n",
-    "out = summation([ratio(a, b) for (a, b) in zip(ina, inb)])\n",
+    "out = summation(*[ratio(a, b) for (a, b) in zip(ina, inb)])\n",
```

Comment:
I don't think the `summation` function as currently written makes a lot of sense.

```python
>>> def summation(*a):
...     return sum(*a)
... 
>>> summation(1, 2, 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 2, in summation
TypeError: sum() takes at most 2 arguments (3 given)
```

Instead, the following feels more intuitive.
```python
>>> def new_summation(*a):
...     return sum(a)
... 
>>> new_summation(1, 2, 3)
6
>>> new_summation(*[1, 2, 3])
6
```

Reply:
How about 
```
def summation(a)
    return sum(a)
```
and then you don't need "*" anywhere?

Reply:
I agree it's cleaner if we don't use `*` at all -- it's more or less noise that takes away from the actual lesson.

I removed the `summation` function altogether in my latest commit in favor of just doing `delayed(sum)`. Let me know what you think.

Reply:
That's certainly what we used to do, the last time I went through this material, so I'm not sure why it changed; probably in the `delayed` version of this it still has the original.